addons/io_hubs_addon/components/definitions/fog.py

```diff
@@ -29,7 +39,7 @@ class Fog(HubsComponent):
                                min=0,
                                max=1)
 
-    # TODO Make these properties to be displayed dynamically based on the fog type
+    # TODO Make these properties to be displayed dynamically based on the fog type, BlenderDiplom: Done
```

Comment:
Probably better to remove this as it's not a TODO anymore and the commit and the commit author already give you the information that you are adding.